packages/fiori-freestyle-writer/templates/listdetail/add/webapp/Component.ts

```diff
@@ -53,7 +53,7 @@ export default class Component extends UIComponent {
     public getContentDensityClass(): string {
         if (this.contentDensityClass === undefined) {
             // check whether FLP has already set the content density class; do nothing in this case
-            // eslint-disable-next-line sap-no-proprietary-browser-api
+            // eslint-disable-next-line fiori-custom/sap-no-proprietary-browser-api
```

Comment:
If we (or our rules) believe we shouldn't do this, why do we have in the template?

Reply:
It has been there for a very long time in JS version. 
Can you suggest an alternative or maybe this should be a separate TBI to update JS and TS templates?

Reply:
Fine with me.